data.py

```diff
@@ -27,7 +28,7 @@ def __init__(self, name, split, transforms, image_key, label_key='label'):
         except ImportError:
             raise ImportError('Please install datasets with `pip install datasets`.')
 
-        self.df = load_dataset(name, split=split)
+        self.df = load_dataset(name, split=split, cache_dir=DATA_DIR)
```

Comment:
would recommend using `export HF_DATASETS_CACHE` in `~/.bashrc` instead of manually setting it here.

https://huggingface.co/docs/datasets/cache#cache-directory

Reply:
fixed in `766e1054e6bf37335ed8c290a2324041bb33178c`